challenges/objects-arrays.js

```diff
@@ -98,14 +167,19 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
+zooAnimals.map(function (currentValue){
+  lowerCase.push(currentValue.animal_name.toLowerCase())});
```

Comment:
Remember that the map() function will change the array and return a new array. The callback function will return the changed item for each index. Look up the MDN docs on .map() and let me know if you are still confused!